### ISSUE TITLE
Infil Test

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#1401]
Hi Robson,

A client sent me this infil data and I set it up for a quick test.  

It's odd because the new green ampt calculator finishes and the old green ampt calculator shows an error.  So maybe that means it's fixable.

https://github.com/user-attachments/assets/080db3e2-ca7f-466d-99bb-e7798a88ec4f

Here's the test data.
https://flo-2d.sharefile.com/d-s71455d3f998d4876b6d9c9c054e28084

**Comment by rpachaly** [comment]
I took me like forever to figure out what was happening. On the calculate_dtheta function, the if statement for dry, saturated or wet and normal was considering ONLY lowercase letters. This client data uses initial uppercase letters (e.g. 'Dry' & 'Normal'). I improved the function by always considering the saturation as lowercase letters. This should not happen again. Done and merged on: https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/pull/1402

**A quick fix for the client is to change Dry to dry and Normal to normal.**
